sdcm/nemesis.py

```diff
@@ -1062,15 +1062,16 @@ def disrupt_no_corrupt_repair(self):
             self._prepare_test_table(ks=f'drop_table_during_repair_ks_{i}', table='standard1')
 
         self.log.debug("Start repair target_node in background")
-        thread1 = threading.Thread(target=self.repair_nodetool_repair, name='NodeToolRepairThread', daemon=True)
-        thread1.start()
-
-        # drop test tables one by one during repair
-        for i in range(10):
-            time.sleep(random.randint(0, 300))
-            with self.cluster.cql_connection_patient(self.target_node) as session:
-                session.execute(f'DROP TABLE drop_table_during_repair_ks_{i}.standard1')
-        thread1.join(timeout=120)
+        with ThreadPoolExecutor(max_workers=1, thread_name_prefix='NodeToolRepairThread') as thread_pool:
+            thread = thread_pool.submit(self.repair_nodetool_repair)
+            try:
```

Comment:
don't you need an except statement here (because of this try)? if not, why do you use this try?

Reply:
no we don't need one:
https://stackoverflow.com/a/11552040/459189